fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/api/event/player/BlockBreakEvents.java

```diff
@@ -70,21 +100,23 @@
 	 * <li> entity - The block entity of the broken block (can be null)
 	 * <li> block - The block instance of the block that was broken</ul>
 	 */
-	public static final Event<AfterBreakBlockCallback> AFTER = EventFactory.createArrayBacked(AfterBreakBlockCallback.class,
-			(listeners) -> (world, player, pos, state, entity, block) -> {
-				for (AfterBreakBlockCallback event : listeners) {
-					event.afterBlockBreak(world, player, pos, state, entity, block);
-				}
-			}
-	);
-
 	@FunctionalInterface
-	public interface BeforeBreakBlockCallback {
-		ActionResult beforeBlockBreak(World world, PlayerEntity player, BlockPos pos, BlockState state, /* Nullable */ BlockEntity entity, Block block);
+	public interface After {
+		void afterBlockBreak(World world, PlayerEntity player, BlockPos pos, BlockState state, /* Nullable */ BlockEntity entity);
 	}
 
+	/* Called when a block break has been canceled
+	 *
+	 * <p>Fields:
+	 * <ul><li> world - The world where the block was going to be broken
+	 * <li> player - The player was going to break the block
+	 * <li> pos - The position where the block was going to be broken
+	 * <li> state - The block state of the block that was going to be broken
+	 * <li> entity - The block entity of the block that was going to be broken (can be null)
+	 * <li> block - The block instance of the block that was going to be broken</ul>
```

Comment:
Please move the fields onto the method javadoc. It's going to be easier to read that way.

Reply:
Done